.github/workflows/filterDuplicates.js

```diff
@@ -0,0 +1,268 @@
+const fs = require('fs/promises')
```

Comment:
1 or 2 sentences are a cheap way to give insight to someone trying to understand this script. Also a USAGE section is good practice.

At a glance I guess something like:

```
// Modifies a `jscpd-report.json` produced by `jscpd`. Removes X based on Y from
// a diff produced by `git diff`.
//
// Usage:
//      node filterDuplicates.js run diff_output.txt jscpd-report.json
// ...
```